specs/electra/beacon-chain.md

```diff
@@ -1470,6 +1505,10 @@ def process_consolidation_request(
         source_index=source_index,
         target_index=target_index
     ))
+
+    # Churn any target excess active balance of target and raise its max
```

Comment:
can we keep this in `process_pending_consolidations`?

so drop this addition here, and then keep the deletion up above

```python
# Churn any target excess active balance of target and raise its max
switch_to_compounding_validator(state, pending_consolidation.target_index)
```

seems easier to reason about if the effects are all in one place

Reply:
I like the idea of switching to compounding in one place. So, the consolidation request has an additional semantics that takes an effect disregarding whether it is a genuine consolidation or merely a switch request. The switch also adds pending balance to the activation queue which creates additional dependency between two process epoch routines: `process_pending_consolidations` and `process_pending_balance_deposits`, having no additional dependency is cleaner.